### PR TITLE
Presets fix: search bar z-index should keep it above the presets panels

### PR DESCRIPTION
**Pull request by limonspb**
PR [#3052](https://github.com/betaflight/betaflight-configurator/pull/3052)  introduced a visual bug when scrolling through presets on PC.
Current:
image

Fixed:
image

### CODE REVIEW

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
verified working - Linux portable - Debian.

**Review by Asizon — APPROVED** [review]

**Comment by limonspb** [comment]
> verified working - Linux portable - Debian.

Thank you!

**Pull request merged**
